pdr_backend/pdr_dashboard/pages/home.py

```diff
@@ -0,0 +1,158 @@
+from dash import html, dcc
+import dash_bootstrap_components as dbc
+from pdr_backend.pdr_dashboard.dash_components.view_elements import (
+    get_table,
+    get_feeds_data,
+    get_predictoors_data,
+    get_metric,
+    get_date_period_selection_component,
+)
+from pdr_backend.pdr_dashboard.dash_components.util import (
+    get_feed_ids_based_on_predictoors_from_db,
+)
+
+
+def layout(app):
```

Comment:
I would suggest converting all of these to a class, now. Since we evolved the knowledge required by each component, I would suggest creating Page/AppPage objects which take in `app` in the constructor. Functions that use knowledge from app or any other common structures can therefore be independent and parameter passing reduced. We can define a light interface using abc that requires all pages to have a layout() function.

In time, the procedural approach will get too complicated to manage so it's better to do this now.